mllib/src/main/scala/org/apache/spark/ml/feature/MinHashLSH.scala

```diff
@@ -31,13 +31,9 @@ import org.apache.spark.sql.types.StructType
 /**
  * :: Experimental ::
  *
- * Model produced by [[MinHash]], where multiple hash functions are stored. Each hash function is
- * a perfect hash function:
- *    `h_i(x) = (x * k_i mod prime) mod numEntries`
- * where `k_i` is the i-th coefficient, and both `x` and `k_i` are from `Z_prime^*`
- *
- * Reference:
- * [[https://en.wikipedia.org/wiki/Perfect_hash_function Wikipedia on Perfect Hash Function]]
+ * Model produced by [[MinHashLSH]], where multiple hash functions are stored. Each hash function is
+ * a perfect hash function for a specific set `S` with cardinality equal to a half of `numEntries`:
```

Comment:
You are right. Actually the correct universal hash function is (ax+b) % p % m (https://en.wikipedia.org/wiki/Universal_hashing#Hashing_integers), which is approximately
min-wise independent families (http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.121.8215&rep=rep1&type=pdf)